tests/unit/amo/reducers/test_versions.js

```diff
@@ -173,4 +195,335 @@ describe(__filename, () => {
       ).toEqual(null);
     });
   });
+
+  describe('load versions for add-ons', () => {
+    const versionId = 99;
+    const version = { ...fakeVersion, id: versionId };
+
+    describe('LOAD_ADDONS_BY_AUTHORS', () => {
+      it('loads versions', () => {
+        const state = versionsReducer(
+          undefined,
+          loadAddonsByAuthors({
+            addons: [
+              {
+                ...fakeAddon,
+                current_version: version,
+              },
+            ],
+            authorUsernames: [fakeAddon.authors[0].username],
+            count: 1,
+            pageSize: DEFAULT_API_PAGE_SIZE,
```

Comment:
This is repeated everywhere and it's not relevant to any of the tests. That's a good reason to make a `_loadAddonsByAuthors` helper with some defaults. It could also calculate `count`.